### PR TITLE
modify PR template to explain that config must be in its own PR

### LINKED ISSUE #832: suggestion: change to PR template
The template should probably explain that *any* modification to configuration or package.json should be in its own PR. This is because it is not intuitively obvious to those who have not been burned by difficulty reverting PRs that config changes should be separate, even ones that are tightly integrated with the PR.

### PR DESCRIPTION
**Pull request by cellog**
# Description

Fixes #832 by adding sub-categories to the PR template like below and slightly adjusting formatting:

```markdown
# Description

*(Delete this, but please include a summary of the change and which issue is fixed -include its number-. Please also include relevant motivation and context.)*

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

PS: [Read how to write the perfect pull request](https://blog.github.com/2015-01-21-how-to-write-the-perfect-pull-request/)
```

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [X] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [X] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

PS: [Read how to write the perfect pull request](https://blog.github.com/2015-01-21-how-to-write-the-perfect-pull-request/)

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
 # Description
 
-(Delete this, but please include a summary of the change and which issue is fixed -include its number-. Please also include relevant motivation and context.)
+*(Delete this, but please include a summary of the change and which issue is fixed -include its number-. Please also include relevant motivation and context.)*
 
 # Checklist:
 
 - [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
+  - [ ] This PR only contains configuration changes (package.json, etc.)
```
Comment (julien51) [inline PULL_REQUEST_TEMPLATE.md]: nit: White space at beginning of line ;) Please align with the rest!

Comment (julien51) [reply PULL_REQUEST_TEMPLATE.md]: Hooooo, now I see that these are probably intentional? Do they actually indent?

**Comment by cellog** [comment]
testing:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [X] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (cellog) [reply PULL_REQUEST_TEMPLATE.md]: it indents, just confirmed :)

**Pull request merged**